### PR TITLE
fix standards pages to show scored evidence activities; update helpful article on activity summary page

### PR DESCRIPTION
**Pull request by emilia-friedberg**
## WHAT
Update the standards page and backend queries to show data for Evidence activities that have been scored.

## WHY
We want these pages to reflect up-to-date data.

## HOW
Add a new attribute called "total_scored_student_count" to use in place of "total_student_count" where relevant, because we don't want to show numbers like "1 of 15" proficient when only 1 of those 15 sessions was actually scored, and update the frontend for both the index and standard-specific page to show that.

### Screenshots
Screenshot 2024-08-29 at 12 05 24 PM
Screenshot 2024-08-29 at 12 05 38 PM

### Notion Card Links
https://www.notion.so/quill/Standards-report-doesn-t-seem-to-be-recognizing-updated-scoring-5a21c0c52b5f42139ffb57debd526d47?pvs=4
https://www.notion.so/quill/Swap-out-helpful-article-on-Activity-Summary-page-0771971d28484098aff4fd463b007871?pvs=4

### What have you done to QA this feature?
Test out on staging to ensure that old Evidence sessions are still represented as unscored, but newer ones show the score.

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  YES
Have you deployed to Staging? | YES
Self-Review: Have you done an initial self-review of the code below on Github? | YES

### CODE REVIEW

File: services/QuillLMS/spec/serializers/progress_reports/standards/standard_serializer_spec.rb [hunk 1]
```diff
@@ -50,7 +50,8 @@
                            average_score
                            standard_students_href
                            mastery_status
```
Comment (brendanshean) [inline services/QuillLMS/spec/serializers/progress_reports/standards/standard_serializer_spec.rb:52]: You could make the indentation less with a different breakdown:

```
    it 'includes the right keys' do
      expect(parsed_classroom.keys)
        .to match_array %w[
          name
          total_student_count
          proficient_student_count
          not_proficient_student_count
          total_standard_count
          students_href
          standards_href
        ]
    end
```

File: services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/standards_standards_progress_report.jsx [hunk 2]
```diff
@@ -117,11 +117,13 @@ export default class StandardsProgressReport extends React.Component {
 
   formatStandardsData(data) {
     return data.map((row) => {
+      const showNotScored = row.is_evidence && row.total_scored_student_count === 0
```
Comment (brendanshean) [inline services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/standards_standards_progress_report.jsx:120]: Nice refactor.

**Review by brendanshean — APPROVED** [review]
LGTM.  Some nice refactoring in the frontend.

**Review by anathomical — APPROVED** [review]

File: services/QuillLMS/spec/serializers/progress_reports/standards/standard_serializer_spec.rb (hunk 1 above)
Comment (emilia-friedberg) [reply services/QuillLMS/spec/serializers/progress_reports/standards/standard_serializer_spec.rb:52]: @brendanshean I actually tried that, but the linter got mad about the brackets not being aligned.

**Pull request merged**